### PR TITLE
Remove delay before PXE service

### PR DESCRIPTION
**Pull request by mdhorn**
Changes proposed in this pull request:
- Extend delay before starting the installer service

### CODE REVIEW

File: etc/systemd/clr-installer-provision.service [hunk 1]
```diff
@@ -19,7 +19,7 @@ Conflicts=getty@tty1.service getty@tty0.service serial-getty@ttyS0.service
 [Service]
 Type=oneshot
 ExecStartPre=/bin/bash -l -c 'grep --quiet --no-message clri.descriptor= /proc/cmdline || (echo "Missing clr-install YAML on kernel command line" ; /bin/false)'
-ExecStartPre=-/bin/bash -l -c '/usr/bin/sleep 10'
+ExecStartPre=-/bin/bash -l -c '/usr/bin/sleep 30'
```
Comment (jwakre) [inline etc/systemd/clr-installer-provision.service]: What happens when the network isn't ready after 10s? Does the until loop still work?

Comment (mdhorn) [reply etc/systemd/clr-installer-provision.service]: _Originally posted by @ionutnechita in https://github.com/clearlinux/clr-installer/issues/247#issuecomment-562874046_
> Hi Clear Linux Team,
> 
> If we can increase the time for clr-installer-provision.service, would help me.
> ExecStartPre=/usr/bin/sleep 10 -> ExecStartPre=/usr/bin/sleep 30
> 
> 1. My DHCP server is slower.
> 2. The interface didn't start up fast for vm using vmware esxi.

@ionutnechita  
John has a good question, I would expect that the "unit" loop to work once network comes up.

Comment (mdhorn) [reply etc/systemd/clr-installer-provision.service]: @ionutnechita Can you share more about the failure with the current sleep at 10?
In the loop that runs until the install is successful, we should be completely restarting the networking stack which I would assume would attempt to get a DHCP address if one was not present?

**Comment by mdhorn** [comment]
Fixed in Clear Installer release [2.3.7](https://github.com/clearlinux/clr-installer/releases/tag/2.3.7)
Available in Clear Linux OS 31930 or later
https://clearlinux.org/downloads

**Pull request merged**